src/Administration/Resources/app/administration/src/module/sw-cms/component/sw-cms-page-form/sw-cms-page-form.scss

```
@@ -169,4 +169,11 @@
         align-items: center;
         justify-content: flex-end;
     }
+
+    &__overwrite-reset {
```

```suggestion
    &__override-reset {
```
Let's stay consistent here.